libs/application/templates/aosh/work-accident-notification/src/fields/AddEmployee/index.tsx

```
@@ -0,0 +1,79 @@
+              employeeAmount: employeeAmount + 1,
+            },
+          },
+          locale,
+        },
+      })
+
+      if (res.data) {
+        setUndefinedEmployeeAmount(false)
+        setCouldNotAddEmployee(false)
+        // Go to screen
+        goToScreen && goToScreen(`employeeInformation[${employeeAmount}]`)
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)